docs/guidelines.md

```diff
@@ -290,29 +297,55 @@ The extra `*` on the multiline comment and `/` on the single line comment are
 important. They inform Doxygen that comment is the documentation for the
 following declaration.
 
-For groups of very similar things you can avoid repeating yourself with `///{`
-and `///}`:
-
-```C++
-/**
- * Get a string
- */
-///{
-char* get_c_string();
-std::string get_cpp_string();
-///}
-```
-
 If referring to something that happens to be a namespace or other global
 object (like DDS, OpenDDS, or RTPS), you should precede it with a `%`.
 If not it will turn into a link to that object.
 
 For more information, see [the Doxygen manual](http://www.doxygen.nl/manual/).
 
-### Preprocessor Macros
+### Preprocessor
+
+- Prefer `#ifdef` and `#ifndef` to `#if defined` and `#if !defined` when
+  testing if a single macro is defined.
+- As stated before, preprocessor macros visible to user code must begin with
+  `OPENDDS_`.
+- Ignoring the header guard if there is one, preprocessor statements should be
+  indented using two spaces starting at the pound symbol, like so:
+
+```C++
+#if defined(X) && defined(Y)
```

Comment:
```suggestion
#if defined X && defined Y
```
Any reason not to?  `defined` is an operator and not a function.

Reply:
No, just the way I'm used to writing it. Do you want me to add that as well since you're asking about it?